deployment/ansible/roles/model-my-watershed.tiler/tasks/configuration.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Configure Node Modules mount
+  template: src=systemd-mount.j2
```

Comment:
I would consider renaming this template file to `systemd-tiler.mount.j2` for consistency with the other unit files.